README.md

```diff
@@ -752,9 +752,9 @@ capability | setting | default value
 
 ##### OCI Defaults: Resource Limits
 
-Each of the `resource-limits` settings below contain two numeric fields: `hard-limit` and `soft-limit`, which are **32-bit unsigned integers**.
+Each of the `resource-limits` settings below contain two fields: `hard-limit` and `soft-limit`, which are **integers between -1 to I64::MAX inclusive** or **unlimited**.
```

Comment:
Is this information repeated in [929](https://github.com/bottlerocket-os/bottlerocket/blob/fb831a78a6a794be82f4930628bd86dc374c3ebe/README.md?plain=1#L929)?

Reply:
Yes, we are repeating  the same information, but line 929 make sense because after that line we are explaining the usage of those values. May be this line chan be changed to just Each of the `resource-limits` settings below contain two fields: `hard-limit` and `soft-limit`

Reply:
> Each of the resource-limits settings below contain two fields: hard-limit and soft-limit

I like it.